### PR TITLE
Update lemmy.yml

### PR DESCRIPTION
**Pull request by DMTryptamines**
https://github.com/LemmyNet/lemmy-ansible/issues/35

### CODE REVIEW

**Comment by Nutomic** [comment]
This doesnt look correct. I think the problem is that `isreg` is only set if the file exists. So the correct condition is probably `reg_enabled.stat.exists and reg_enabled.stat.isreg`.

cc @fadelkon

**Comment by fadelkon** [comment]
El 11 d’abril de 2022 14:10:40 CEST, Nutomic ***@***.***> ha escrit:
>This doesnt look correct. I think the problem is that `isreg` is only set if the file exists.

I see

>  So the correct condition is probably `reg_enabled.stat.exists and reg_enabled.stat.isreg`.
>
>cc @fadelkon 
>

Sure, makes sense!

File: lemmy.yml [hunk 1]
```diff
@@ -81,7 +81,7 @@
         file:
           path: "/etc/nginx/sites-enabled/{{domain}}.conf"
           state: absent
-        when: reg_enabled.stat.isreg
```
Comment (dessalines) [inline lemmy.yml:84]: reg_enabled.stat.exists and reg_enabled.stat.isreg

**Review by dessalines — CHANGES_REQUESTED** [review]

**Review by DMTryptamines — COMMENTED** [review]
Tested and can confirm r`eg_enabled.stat.exists and reg_enabled.stat.isreg` completed a fresh install fine where 'reg_enabled.stat.isreg` alone did not.

**Comment by dessalines** [comment]
Thanks!

**Pull request merged**